### PR TITLE
BIN-82: Tool runs with errors

### PR DESCRIPTION
**Pull request by liodnik**

### CODE REVIEW

File: mysql_merge/config.py.example [hunk 1]
```diff
@@ -106,4 +106,9 @@ default_mapping = { }
 """
 enum_tables = []
 
+"""
+  List of tables with their's ids that should be ignored during merge.
+"""
+not_increment = {'table_name': [1, 2]}
```
Comment (mikedld) [inline mysql_merge/config.py.example]: Rename to `ids_to_ignore`

File: mysql_merge/config.py.example [hunk 2]
```diff
@@ -106,4 +106,9 @@ default_mapping = { }
 """
 enum_tables = []
```
Comment (mikedld) [inline mysql_merge/config.py.example]: Rename to `tables_to_ignore`

File: mysql_merge/mysql_merger.py [hunk 3]
```diff
@@ -336,96 +347,26 @@ def copy_data_to_target(self):
...
                 else:
-                    self._cursor.execute("INSERT IGNORE INTO `%(destination_db)s`.`%(table)s` (%(columns)s) SELECT %(columns)s FROM `%(source_db)s`.`%(table)s` %(where)s" % {
-                        'destination_db': self._destination_db['db'],
-                        'source_db': self._source_db['db'],
-                        'table': table_name,
-                        'where': where,
-                        'columns': "`%s`" % ("`,`".join(columns))
-                    })
+                    self._cursor.execute(
+                        "INSERT IGNORE INTO `%(destination_db)s`.`%(table)s` (%(columns)s) SELECT %(columns)s FROM `%(source_db)s`.`%(table)s` %(where)s" % {
```
Comment (iulasenka) [inline mysql_merge/mysql_merger.py]: don't we want to use just insert here as it was proposed in other script?

**Review by iulasenka — APPROVED** [review]
Looks good except minor note above

File: mysql_merge/mysql_merger.py (hunk 3 above)
Comment (liodnik) [reply mysql_merge/mysql_merger.py]: Sure, let me change INSERT for this case

**Pull request merged**